scuba/__main__.py

```diff
@@ -69,9 +70,18 @@ def parse_scuba_args(argv):
     ap.add_argument('-v', '--version', action='version', version='scuba ' + __version__)
     ap.add_argument('-V', '--verbose', action='store_true',
             help="Be verbose")
+
+    def list_aliases_completer(**_):
+        try:
+            _, _, config = ScubaDive.get_paths_and_config()
+            return sorted(config.aliases)
+        except (ConfigNotFoundError, ConfigError):
+            return []
```

Comment:
I guess this probably fixes #101?

Would this be a usecase for [`argcomplete.warn`](https://kislyuk.github.io/argcomplete/#printing-warnings-in-completers)?

Reply:
Yes (to fixes #101).

It it truly a warning to run without a `.scuba.yml`?  It seems like development is moving in the direction of that being a "normal" use case.

Reply:
Well, it's not warning-worthy to simply "run" scuba without `.scuba.yml`. But follow this logic:

- If you type `scuba <TAB>`, you're asking scuba to assist in completing your command line.
- Since you haven't provided anything for `args.command` yet, you must be looking for a "command" (and not args to that command)
- Since scuba can't know the available commands inside the image, the only thing it can offer are aliases.
- But it can't do that if there isn't a `.scuba.yml`.

So I think in the case where the user has provided nothing to `command` yet, then this is a warning.

And this conversation made me realize the we should really look at what's been provided to `command`. You currently have `def list_aliases_completer(**_):` but I'm assuming there is interesting context being passed to the completer.
- If you type `scuba anything <TAB>` (you've provided at least one thing to `args.command`), then scuba should not attempt to complete anything. In fact, we should *cancel* our completion (is that a thing?) and let bash go back to the normal behavior, suggesting files in the current directory.

Reply:
That makes sense.

There is [context](https://kislyuk.github.io/argcomplete/#specifying-completers) with completers that we can use to do what you're suggesting.  To "cancel a completion", it looks like returning an empty list is sufficient.

Reply:
Addressed in TimothySprague/scuba@f48e2fd5